### DISCUSSION TITLE
DFN model for a multi-layered pouch cell

### DISCUSSION THREAD

**Discussion by abhishek-appana** [pybamm-team/PyBaMM#4010] [Q&A] [answered]
Hello everyone,

I need to use a DFN model for a multi-layered pouch cell (25 layers). I see that the PyBaMM DFN model currently models a single-layer pouch cell by default. I read in older discussions that the easiest way to do a multi-layered cell right now is to increase the cell dimensions. Sadly, that approach won't work for me because I need to use the current collector and thermal balance submodels along with DFN. Changing the dimensions would affect the whole simulation.

What is the hard but feasible way of doing this? Is there any example that someone developed previously, that I can use as a reference? If anyone has any ideas or suggestions, please chime in!

Thank you!

**Comment by rtimms** [comment]
Which thermal model would you like to use? If you want a lumped model then the easiest thing to do is change the parameter “Number of electrodes connected in parallel to make a cell"  to give the right number of electrode pairs, and then use a lumped thermal model with an arbitrary geometry updating the parameters “Cell cooling surface area [m2]" and "Cell volume [m3] to give the correct dimensions.

**Reply by abhishek-appana** [reply]
Hello @rtimms,

Thank you for your response!

I want to use the 2+1D pouch cell thermal model. Will the above changes suffice or do I need to make any other changes?

**Reply by rtimms** [reply, marked as answer]
See https://github.com/pybamm-team/PyBaMM/issues/1777

**Reply by abhishek-appana** [reply]
Perfect! Thank you so much 😀
